lib/AST/Decl.cpp

```diff
@@ -4206,6 +4205,14 @@ bool ValueDecl::hasOpenAccess(const DeclContext *useDC) const {
   return access == AccessLevel::Open;
 }
 
+bool ValueDecl::bypassResilienceInPackage(ModuleDecl *accessingModule) const {
+  return getASTContext().LangOpts.EnableBypassResilienceInPackage &&
+          getModuleContext()->inSamePackage(accessingModule) &&
+          !getModuleContext()->isBuiltFromInterface() &&
+          getFormalAccessScope(/*useDC=*/nullptr,
+                               /*treatUsableFromInlineAsPublic=*/true).isPackage();
```

Comment:
I think you don't need the isPackage() check -- it should be safe to access public decls non-resiliently too across a package boundary.

Reply:
I'll push a separate PR for that.